Zh3r0_CTF_V2_2021-Real_Mersenne/server_files/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:20.04
+RUN export DEBIAN_FRONTEND=noninteractive && apt update -y && apt -y install xinetd python3 python3-pip
+
+RUN useradd ctf
+
+WORKDIR /app
+
+RUN chown ctf /app
+
+COPY challenge.py secret.py .
```

Comment:
`ERROR: Service 'real_mersenne-zh3r0_ctf_v2202' failed to build: When using COPY with more than one source file, the destination must be a directory and end with a /`